sympy/physics/control/lti.py

```diff
@@ -3838,6 +3840,108 @@ def num_outputs(self):
         """
         return self._D.rows
 
+    def state_vector(self, initial_conditions=None, input_vector=None, var=None):
```

Comment:
Given these are all kwargs, what happens if you call `.state_vector()`. I don't see a test case for that.